curriculum/challenges/english/10-coding-interview-prep/rosetta-code/cramers-rule.md

```diff
@@ -8,9 +8,9 @@ dashedName: cramers-rule
 
 # --description--
 
-<a href="https://rosettacode.org/wiki/Cramer%27s_rule" target="_blank" rel="noopener noreferrer nofollow">Cramer's rule</a> is a formula for solving a system of linear equations by using the determinants of matrixes formed from subsets of the coefficients and right hand side values.
+<a href="https://rosettacode.org/wiki/Cramer%27s_rule" target="_blank" rel="noopener noreferrer nofollow">Cramer's rule</a> is a formula for solving a system of linear equations by using the determinants of matrices formed from subsets of the coefficients and right hand side values.
```

Comment:
I am wondering, what is the difference between matrixes and matrices?

Reply:
Matrices is the correct plural form of matrix; also the rest of the doc used matrices and not matrixes, so I changed it for consistency